ui/desktop/app/templates/scopes/scope/projects/targets/index.hbs

```diff
@@ -13,189 +13,149 @@
-    {{#if
-      (or
-        @model
-        (has-resource-filter-selections
-          'scopes.scope.projects.targets' 'project'
-        )
-      )
-    }}
-      <Rose::Toolbar as |toolbar|>
-        <toolbar.primary>
-          <Rose::Form as |form|>
-            {{#let
-              (resource-filter 'scopes.scope.projects.targets' 'project')
-              as |filter|
-            }}
-              <Rose::Dropdown
-                @text={{t (concat 'resources.' filter.name '.title')}}
-                @count={{filter.selectedValue.length}}
-                @ignoreClickInside={{true}}
-                as |dropdown|
-              >
-                <ul>
-                  {{#let
-                    (group-by filter.allowedValues 'scopeModel')
-                    as |groups|
-                  }}
-                    {{#each groups as |groupedItems|}}
-                      <li>
-                        <dropdown.item>
-                          <Rose::Icon
-                            @name='flight-icons/svg/org-16'
-                            @size='medium'
-                          />
-                          {{groupedItems.key.displayName}}
-                        </dropdown.item>
-                        <ul class='indent-label-2'>
-                          <form.checkboxGroup
-                            @name={{filter.name}}
-                            @items={{groupedItems.items}}
-                            @selectedItems={{filter.selectedValue}}
-                            @onChange={{route-action 'filterBy' filter.name}}
-                            as |group|
-                          >
-                            <li>
-                              <dropdown.item>
-                                <group.checkbox
-                                  @label={{group.item.displayName}}
-                                  value={{group.item}}
-                                />
-                              </dropdown.item>
-                            </li>
-                          </form.checkboxGroup>
-                        </ul>
-                      </li>
-                    {{/each}}
-                  {{/let}}
-                </ul>
-              </Rose::Dropdown>
-            {{/let}}
-
-            {{#if
-              (has-resource-filter-selections
-                'scopes.scope.projects.targets' 'project'
-              )
-            }}
-              <Rose::Button
-                @style='inline-link-action'
-                @iconLeft='flight-icons/svg/x-16'
-                {{on 'click' (route-action 'clearAllFilters')}}
-              >{{t 'actions.clear-all-filters'}}
-              </Rose::Button>
-            {{/if}}
-          </Rose::Form>
-        </toolbar.primary>
-
-        <toolbar.secondary>
-          <ToolbarRefresher @onClick={{route-action 'refreshTargets'}} />
-        </toolbar.secondary>
-
-      </Rose::Toolbar>
-    {{/if}}
-
+  <page.body class='targets'>
     {{#if @model}}
```

Comment:
Why do we have a check for `@model` here but `this.targets` below? It seems like these can fall out of sync now?